skills_ml/evaluation/embedding_metrics.py

```diff
@@ -0,0 +1,142 @@
+from skills_ml.ontologies.clustering import Clustering
+
+from collections import defaultdict
+from scipy.spatial import distance
+from gensim.models import KeyedVectors
+from typing import Callable, Dict
+import numpy as np
+import tempfile
+import os
+
+
+def embedding_evaluate(vectorization_pipeline, metric_objects):
```

Comment:
I think we should name this to mirror the skill extractor evaluation routines: `metrics_for_embedding`